app/src/lib/stores/app-store.ts

```diff
@@ -3750,7 +3778,6 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
   private async withPushPullFetch(
     repository: Repository,
-    account: IGitAccount | null,
```

Comment:
I just stumbled upon the fact that this parameter was unused so I removed it which let me keep a newly introduced call to this method on one line. It's unrelated to this PR but it's small enough that I'm hoping we can just sneak it in.